styles.css

```diff
@@ -1,504 +1,1092 @@
+  body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    background-color: #e5e5e5;
+  }
+
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+
+    /* margin: 0px 60px; */
+
+    width: 100%;
+    background-color: #fff;
+  }
+
+  #logo {
+    text-decoration: none;
+    width: 106px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 25px;
+    line-height: 20px;
+    color: #6070ff;
+    margin-left: -12%;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .top_navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    width: 258px;
+    height: 40px;
+    gap: 8px;
+    padding-left: 30px;
+    position: relative;
+    left: 40%;
+  }
+
+  .navbar_li {
+    width: 62px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  /* main-section */
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 800px;
+    padding-top: 88px;
+    background: #fff;
+    background-image: url(./images/Header\ bg@2x.png);
+    background-size: 100%;
+    background-repeat: no-repeat;
+    background-position-y: 60px;
+    background-position-x: center;
+    border-bottom-left-radius: 100px;
+  }
+
+  .primary-text {
+    max-width: 470px;
+    height: 122px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    line-height: 60px;
+    margin-top: 200px;
+    margin-left: 17%;
+  }
+
+  .paragraph {
+    max-width: 72%;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin: -25px 30px -8px 17%;
+  }
+
+  .connect {
+    width: 117px;
+    height: 24px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-top: -25px;
+    margin-left: 17%;
+    text-decoration: none;
+  }
+
+  .social {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 170.17px;
+    height: 20px;
+    list-style: none;
+    padding-left: 30px;
+    position: relative;
+    bottom: 20px;
+    margin-left: 15%;
+  }
+
+  .meta {
+    margin-top: 4px;
+  }
+
+  /* work section */
+
+  .works {
+    width: 100%;
+    background-color: #e5e5e5;
+    display: grid;
+
+    /* grid-template-rows: 1fr 1fr; */
+
```

Comment:
- [OPTIONAL] Please Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 😄